tests/adapter/mongo/test_variant_loader.py

```diff
@@ -18,11 +16,19 @@ def test_load_variants_high_treshold(real_populated_database, case_obj):
     ## GIVEN a populated database and a case
     adapter = real_populated_database
     ## WHEN Trying to load variants where no one are above the rank score treshold
-    vcf = VCF(case_obj["vcf_files"]["vcf_sv"])
+    variants = VCF(case_obj["vcf_files"]["vcf_sv"])
+    nr_variants = sum(1 for _ in variants)
+    variants = VCF(case_obj["vcf_files"]["vcf_sv"])
```

Comment:
Normally IO-bound, but with a big db and many genes to check you are probably right it doesn't matter.